server/applicationPattern/applicationPattern/onfModel/services/LogicalTerminationPointWithMappingServices.js

```diff
@@ -597,6 +625,64 @@ function findAndUpdateLogicalTerminationPointInstanceGroupAsync(logicalTerminati
     });
 }
 
+/**
+ * @description This function configures the existing logical-termination-point to the latest values.
+ * Also incase if the tcp,operation client are not available it will be created.
+ * @param {String} logicalTerminationPointConfigurationInput : is an instance of
+ * logicalTerminationPoint/ConfigurationInput class
+ * @return {Promise} object {LogicalTerminationPointConfigurationStatus}
+ **/
+function findAndUpdateLogicalTerminationPointInstanceGroupExcludingOldReleaseAndNewReleaseAsync(logicalTerminationPointConfigurationInput) {
```

Comment:
This function is a copypaste from findAndUpdateLogicalTerminationPointInstanceGroupAsync with the only difference being how the httpClientUuid is found. Make httpClientUuid a parameter to a method that contains the logic of findAndUpdateLogicalTerminationPointInstanceGroupAsync. 

```
findAndUpdateLogicalTerminationPointInstanceGroupAsync {
    let httpClientUuid = await httpClientInterface.getHttpClientUuidAsync(
                applicationName,
                releaseNumber
   );
    findAndUpdateLtp(httpClientUuid);
}

findAndUpdateLogicalTerminationPointInstanceGroupExcludingOldReleaseAndNewReleaseAsync {
  let httpClientUuid = await httpClientInterface.getHttpClientUuidExcludingOldReleaseAndNewRelease(
                applicationName,
                releaseNumber
            );
     findAndUpdateLtp(httpClientUuid);
}
```

Reply:
Thanks for the proposal Dana. And thanks Vanitha for implementing it.